### DISCUSSION TITLE
How to change route prefix ?

### DISCUSSION THREAD

**Discussion by realtebo** [Laravel-Backpack/community-forum#623] [Q&A (Help)] [answered]
I am grouping routes by role

    Route::group([
        'middleware' => [
            'can:access-manager-routes',
            'must.change.password',
        ],
        'prefix' => 'manager',
        'namespace'  => '\App\Http\Controllers\Manager',
    ], function () {

        Route::crud('password', 'PasswordCrudController');

        Route::crud('manager', 'ManagerCrudController');

    ...

Note: prefix' => 'manager', 

The problem is: actually Route::crud are creating routes IGNORING the route group prefix .

```
  GET|HEAD  manager/password/password-change-initial-password ............................................. password.passwordChangeInitialPassword › Manager\PasswordCrudController@passwordChangeInitialPassword
```

I need `manager/password/password-change-initial-password` to have name `manager.password.passwordChangeInitialPassword`

How can I override route prefix in the crud controller ?

**Comment by promatik** [comment, marked as answer]
Hi @realtebo!

The prefix is working, the `manager` prefix is being added to the route;
**`manager/`**`password/password-change-initial-password`

What you want is the prefix for the `name`, and for that you use the `name` attribute on the route group;

```php
Route::group([
	'name' => 'manager.'
    'prefix' => 'manager',
    'namespace'  => '\App\Http\Controllers\Manager',
    ...
```

You can check the documentation over here; https://laravel.com/docs/10.x/routing#route-group-name-prefixes
